server/src/services/boards/columns.go

```diff
@@ -8,23 +8,37 @@ import (
 	"github.com/google/uuid"
 	"scrumlr.io/server/common"
 	"scrumlr.io/server/common/dto"
+	"scrumlr.io/server/common/filter"
 	"scrumlr.io/server/realtime"
 
 	"scrumlr.io/server/database"
 	"scrumlr.io/server/logger"
 )
 
-func (s *BoardService) CreateColumn(_ context.Context, body dto.ColumnRequest) (*dto.Column, error) {
+func (s BoardService) CreateColumn(_ context.Context, body dto.ColumnRequest) (*dto.Column, error) {
 	column, err := s.database.CreateColumn(database.ColumnInsert{Board: body.Board, Name: body.Name, Color: body.Color, Visible: body.Visible, Index: body.Index})
+	if err != nil {
+		logger.Get().Errorw("unable to create column", "err", err)
+	}
```

Comment:
Shouldn't we return here?
Since the db called failed there are no updates to send

Reply:
True, completely missed the return calls in this file, my bad.